R/utils_build_cards.R

```diff
@@ -238,13 +238,17 @@ build_dep_cards <- function(data, loaded, toggled){
     succ_icon = character(),
     icon_class = character(),
     is_perc = numeric(),
-    is_url = numeric()
+    is_url = numeric(),
+    type = "information"
```

Comment:
```suggestion
    type = character()
```

Since all the other values are empty, `dplyr::tibble` is forcing this to `character()` anyways, but I think we should have it coded as an empty value anyways.